src/app/demos/route-metadata/route-order-test/a/index.html

```diff
@@ -1,5 +1,6 @@
 <stache
   pageTitle="a"
-  navOrder="9999">
+  navOrder="9999"
+  hideFromSidebar="true">
```

Comment:
As we will be extending this flag to also keep the pages from being published to the search service or discoverable at all, would calling this something like `publish: false` or `unpublished: true`?  Something along those lines that works for more than just hiding it from the sidebar?

Reply:
in Stache1 we had two options: 
- showInNav: true or false (because you may want it published but not in navigation)
- published: true or false (this enables you to have a page that you want to include in your PR, but not be live yet). Branching may be better for `published`, but sometimes it could be easier for multiple people to work on the same unpublished page and not in a branch.

Reply:
Good points, I'd agree with calling it `showInNav`, and for the time being encourage the use of branches for publishing.